go/test/endtoend/utils/mysql.go

```diff
@@ -114,7 +114,7 @@ func createInitSQLFile(mysqlDir, ksName string) (string, error) {
 		return "", err
 	}
 	defer f.Close()
-
+	_, _ = f.WriteString("SET GLOBAL super_read_only='OFF';")
```

Comment:
Why would we not look for and handle an error here?

Reply:
Isn't this what the init_db.sql file is supposed to do?

Reply:
Should the below line still be here?

Reply:
In these test we create Mysql without any vttablets and then try to create database so I have to set super_read_only to false given its not going through flow of typical vttablet.